baseframe/templates/baseframe/components.html

```diff
@@ -370,12 +208,15 @@
   {%- endif %}
 {%- endmacro %}
 
-{% macro add_hasgeek_auth() %}
+{% macro add_hasgeek_auth(links) %}
   {% if g.user -%}
     <a href="#" data-toggle="dropdown" class="dropdown-toggle" id="hg-user-btn">
       <i class="fa fa-user"></i><b class="caret"></b>
     </a>
     <ul class="dropdown-menu pull-right">
+      {%- for link in links %}
+        <li><a href="{{ link.url }}" {%- if link.rel %} rel="{{ link.rel }}"{% endif %} {%- if link.class %} class="{{ link.class }}"{% endif %}>{{ link.title }}</a></li>
+      {%- endfor %}
```

Comment:
This assumes links are only visible if logged in. Is there a use case for non-logged in links?